pkg/serviceprovider/quay/quay.go

```diff
@@ -240,8 +243,10 @@ func (q *Quay) CheckRepositoryAccess(ctx context.Context, cl client.Client, acce
 		case http.StatusUnauthorized, http.StatusForbidden:
 			lg.Info("quay.io request unauthorized. Probably private repository for we don't have a token.")
```

Comment:
"Probably a private repository for which we don't have a token."

Reply:
fixed